test/getMetafieldsForCheckout.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai')
+const getMetafieldsForCheckout = require('../lib/getMetafieldsForCheckout')
+
+describe('getMetafieldsForCheckout', () => {
+  let globalWindowBackup
+
+  before(() => {
+    globalWindowBackup = global.window
+  })
+
+  after(() => {
+    global.window = globalWindowBackup
+  })
```

Comment:
It's not needed currently as nothing else modifies the window in the tests, but I thought it could be useful to avoid accidental side-effects in the future